### ISSUE TITLE
Add leadership PACs to about this candidate section of candidate profile pages

### ISSUE THREAD

**Issue by JonellaCulmer** [fecgov/fec-cms#4270]
**What we're after:**
We need to add leadership PACs to applicable candidate profile pages. Specifically at the bottom of the about this candidate section. Leadership PAC names should link to those committee profile pages, just as they do for all other committees listed in that Committees section. 

### Related issues
- https://github.com/fecgov/fec-cms/issues/4253 Design for the inclusion of leadership PACs on sponsoring candidate profile pages

### Action items
- [x] Clickable links to leadership PACs are added to applicable candidate profile page "about this candidate" sections
- [x] Glossary links are added to each of the committee subheaders: 
  - Authorized campaign committees: Authorized committee 
  - Joint fundraising committees: Joint fundraising committee
  - Sponsored leadership PACs: Leadership PAC

### Completion criteria
- [x] Leadership PAC committee(s) appear on the candidate profile page under the "About this candidate" tab.
- [x] Open follow-up ticket to add links from committee profile pages to candidate profile pages. Currently it's only name and no link to profile page.

### Design
<img width="779" alt="Screen Shot 2020-12-17 at 4 05 36 PM" src="https://user-images.githubusercontent.com/31663028/102543793-d09f6d00-4081-11eb-9d9d-cdfc96888492.png">

**Comment by fec-jli** [comment]
To display leadership PAC names on candidate profile page, we need a filter by sponsor_candidate_id in endpoint: /candidate/<string:candidate_id>/committees/history/<int:cycle>/
blocked by : [https://github.com/fecgov/openFEC/issues/4738](https://github.com/fecgov/openFEC/issues/4738)

**Comment by patphongs** [comment]
Moving this forward since it's currently blocked by additional endpoint work as stated in the comment above.
